modules/cert-manager-acme-dns01-aws.adoc

```diff
@@ -0,0 +1,113 @@
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have set up an IAM role for Amazon Route 53. For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/route53/[Route53] in the upstream cert-manager documentation.
++
+[NOTE]
+====
+For the {cert-manager-operator}, you must provide an `accessKeyID` and `secretAccessKey` to specify the credentials.
+====
+
+.Procedure
+
+. Configure the nameserver settings for the DNS01 self check:
+
+.. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+.. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig:                                <1>
+    overrideArgs:
+      - '--dns01-recursive-nameservers=1.1.1.1:53' <2>
+      - '--dns01-recursive-nameservers-only'       <3>
+----
+<1> Add the `spec.controllerConfig` section.
+<2> Provide a comma-separated list of `<host>:<port>` nameservers to query for the DNS01 self check.
+<3> Specify to only use recursive nameservers.
++
+// TODO: what exactly does this mean? That it uses caching instead of checking the authoritative nameservers every time? Might cause it to take longer it says why? (every time)?
++
+// TODO: check if v1alpha1 is the correct version?
++
+// TODO Find out if that is just an example value, where they need to replace?
+
+.. Save the file to apply the changes.
+
+. Create an issuer.
+
+.. Create a YAML file that defines the `Issuer` object:
++
+.Example `issuer.yaml` file
+[source,yaml]
+----
+  apiVersion: cert-manager.io/v1
+  kind: Issuer
+  metadata:
+    name: letsencrypt-staging                                        <1>
+    namespace: sandbox                                               <2>
+  spec:
+    acme:
+      server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+      email: "thn@redhat.com"                                        <4>
+      privateKeySecretRef:
+        name: letsencrypt-staging                                    <5>
+      solvers:
+      - dns01:
+          route53:
+            accessKeyID: AKIAT5TK4ZONNAZQMGOD                        <6>
+            hostedZoneID: Z3URY6TWQ91KVV                             <7>
+            region: us-east-1
+            secretAccessKeySecretRef:
+              name: "aws-secret"                                     <8>
+              key: "awsSecretAccessKey"                              <9>
+        selector:
+          dnsNames:
+          - 'example.org'                                            <10>
+----
+<1> TODO
+<2> Specify the namespace for the CA issuer.
```

Comment:
@thejasn @swghosh The example I copied from used "sandbox" as the example namespace. Two questions:

1. Do we need a step in here to create that namespace first?
2. Any suggestions on a better example namespace to use other than sandbox?

Reply:
> Do we need a step in here to create that namespace first?
Yes, you can add it as a prerequisite or have an explicit step with `oc new-project sandbox`

> Any suggestions on a better example namespace to use other than sandbox?
Honestly the namespace isn't very significant, I'm ok with it being sandbox or if there is some sort of convention in OCP docs we can go with that as well.